diff --git a/drivers/bluetooth/hci/Kconfig.infineon b/drivers/bluetooth/hci/Kconfig.infineon
index 92b7f29f8fa..c09ab1db4d0 100644
--- a/drivers/bluetooth/hci/Kconfig.infineon
+++ b/drivers/bluetooth/hci/Kconfig.infineon
@@ -190,6 +190,35 @@ config CYW20829_BT_FW_ISOC_TX10
 endchoice
 endif # CYW20829
 
+choice BT_CYW55500_MODULE
+	prompt "CYW55500 BT module"
+	depends on CYW55500 && BT_H4
+
+config BT_CYW55513IUBG_SM
+	bool "CYW55513 Module"
+	help
+	  AIROC CYW55513 Wi-Fi & Bluetooth is a low-power, single chip device
+	  that supports single-stream, tri-band Wi-Fi 6/6E (IEEE 802.11ax)
+	  Wi-Fi MAC/baseband/radio, and Bluetooth/Bluetooth Low Energy 5.4.
+
+config BT_CYW55513_MURATA_2FY
+	bool "CYW55513 Murata 2FY Module"
+	help
+	  Type 2FY is a small and high performance module based on Infineon
+	  CYW55513 combo chipset which supports Wi-Fi 802.11a/b/g/n/ac/ax +
+	  Bluetooth 5.4 BR/EDR/LE.
+
+	  Detailed information about Murata Type 2FY module can be found on
+	  https://www.murata.com/en-us/products/connectivitymodule/wi-fi-bluetooth/overview/lineup/type2fy
+endchoice
+
+config BT_CYW555XX
+	bool "Bluetooth support for CYW555XX"
+	default y if BT_CYW55513_MURATA_2FY || BT_CYW55513IUBG_SM
+	help
+	  Common Bluetooth configuration for CYW555XX-based designs,
+	  including Murata 2FY modules and discrete chip implementations.
+
 config AIROC_CUSTOM_FIRMWARE_HCD_BLOB
 	string  "Path to user BT firmware HCD file"
 	help
