diff --git a/modules/hal_infineon/btstack-integration/CMakeLists.txt b/modules/hal_infineon/btstack-integration/CMakeLists.txt
index 8033fe30443..36332bf0764 100644
--- a/modules/hal_infineon/btstack-integration/CMakeLists.txt
+++ b/modules/hal_infineon/btstack-integration/CMakeLists.txt
@@ -87,6 +87,16 @@ if(CONFIG_CYW20829_BT_FW_ISOC_TX10)
   set(blob_hcd_file  ${hal_blobs_dir}/COMPONENT_CYW20829B0/COMPONENT_BTFW-ISOC-TX10/bt_firmware.hcd)
 endif()
 
+# CYW55513 Discrete Module
+if(CONFIG_BT_CYW55513IUBG_SM)
+  set(blob_hcd_file  ${hal_blobs_dir}/COMPONENT_CYW55513/COMPONENT_BTFW/bt_firmware.hcd)
+endif()
+
+# CYW55513 Murata 2FY Module
+if(CONFIG_BT_CYW55513_MURATA_2FY)
+  set(blob_hcd_file  ${hal_blobs_dir}/COMPONENT_CYW55513/COMPONENT_BTFW_MURATA-2FY/bt_firmware.hcd)
+endif()
+
 # use user provided FIRMWARE HCD file (path must be defined in CONFIG_AIROC_CUSTOM_FIRMWARE_HCD_BLOB)
 if(CONFIG_AIROC_CUSTOM_FIRMWARE_HCD_BLOB)
   # Allowed to pass absolute path to HCD blob file, or relative path from Application folder.
